### PR TITLE
Update samples to announce UI changes to narrator

### LINKED ISSUE #169: Narrator remains silent after activating Custom XamlUICommand app bar button in XamlUICommand.
**Describe the bug**

**Pre-requisite:** 
Turn on Narrator (Keyboard Shortcut: Ctrl+Win+Enter).
Set Narrator to item mode.

**Steps to reproduce the bug**

1.Launch the XAML Controls Gallery Application.
2. Activate XamlUICommand from All Controls Page.
3. Navigate to "Custom XamlUICommand" app bar button and activate it and observe narrator reading.

**Expected behavior**

After activating the "Custom XamlUICommand" narrator should give the information.

**Actual behavior**

Narrator remains silent without reading anything.

**Screenshots**

image

**Version Info**

Gallery version (found on Settings page): 

Version Info: 1.2.6.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/20822597)

### LINKED ISSUE #334: Narrator doesn't announce the suggestions if it matches with any entry in the Filter By...textboxes in ListView.
**Describe the bug**

**Steps to reproduce the bug**

Prerequisite: 
* Turn on Narrator.
* Keep narrator in item mode.

Repro steps :
1. Launch XAML Controls Gallery.
2. Activate ListView from home page.
3. Navigate to "Filter By" textboxes under the header "ListView with filtering" and enter any data in the textboxes and observe the narrator's description.

**Expected behavior**

Narrator should announce the suggestions if it matches with any entry in the Filter By...textboxes.

**Actual behavior**

Narrator does not announce the suggestions available.

**Screenshots**

image

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/24662739)

### PR DESCRIPTION
**Pull request by marcelwgn**
## Description

This PR adds UIA announcements to samples where they were missing. Also cleaning up unused function in XAMLUICommand page

## Motivation and Context

Closes #169 and closes #334
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/XamlUICommandPage.xaml [hunk 1]
```diff
@@ -28,8 +28,7 @@
                               XamlSource="XamlUICommand\XamlUICommandSample1_xaml.txt"
                               CSharpSource="XamlUICommand\XamlUICommandSample1_cs.txt"
                               HorizontalContentAlignment="Stretch"
-                              MinimumUniversalAPIContract="7"
-                              Loaded="ControlExample_Loaded">
+                              MinimumUniversalAPIContract="7">
```
Comment (karkarl) [inline WinUIGallery/ControlPages/XamlUICommandPage.xaml:31]: What is this change for? Also we should be able to remove any UniversalAPI checks for WinUI3

Comment (marcelwgn) [reply WinUIGallery/ControlPages/XamlUICommandPage.xaml:31]: Just some dead code cleanup. If it's better I can also revert this. My thought was though I'm not sure when someone might stumble across this dead code so remove it now before it stays for the next months or even years.

**Comment by karkarl** [comment]
Can you try rebasing from main? Perhaps this will help with the failed tests

**Review by karkarl — APPROVED** [review]

**Pull request merged**